### ISSUE TITLE
$header variable is declared as a string, but later on treated as an array. This produces a fatal error when sending an email copy with a sender value set from admin.

### ISSUE THREAD

**Issue by ironland** [libreform/wp-libre-form#129]
When the setting "send a copy via email" is selected from admin and the sender email field anything other than the default value, the code in file __wplf-form-actions.php__ tries to append the sender to headers like this __$headers[] = "From: $from <$from_address>";__. The variable is declared as a string, though, and that produces a fatal error: 
__"[] operator not supported for strings"__

The erroneous code block is this:
```
    // default pre-filtered values for email headers and attachments
    $headers = '';
    $attachments = array();

    if ( isset( $form_meta['_wplf_email_copy_from'][0] ) ) {
      $from = wplf_email_copy_replace_tags( $form_meta['_wplf_email_copy_from'][0], $form, $submission_id );
      $from_address = wplf_email_copy_replace_tags( $form_meta['_wplf_email_copy_from_address'][0], $form, $submission_id );
      $headers[] = "From: $from <$from_address>";
    }
```

**Comment by k1sul1** [comment]
Can you post the form configuration as well? This looks like a clear error in the code but just to make sure.

**Comment by ironland** [comment]
Sry for the long wait. Conf from the admin and/or something else?

**Comment by k1sul1** [comment]
Screenshot from the form email settings is fine. I recall using the from field successfully at some point, but I can't be sure as I've used this plugin in dozens of sites, but I currently only have access to one or two of them.

**Comment by ironland** [comment]
![email_settings](https://user-images.githubusercontent.com/6129903/50479175-e8394300-09dd-11e9-81d9-107fdd1c75c7.png)

Here are the email settings edited with test values.

**Comment by ironland** [comment]
I also made a PR with the minor change that fixes this issue. 
https://github.com/libreform/wp-libre-form/compare/master...ironland:patch-1

**Comment by timiwahalahti** [comment]
Merged #146 on commit 8bccce6
